src/Api/Authentication/signIn/signIn.jsx

```diff
@@ -0,0 +1,11 @@
+import axios from 'axios';
+
+async function signIn(postInfo) {
+  const { data } = await axios({
+    url: `${process.env.REACT_APP_LOCAL}Api/Authentication/signUp`,
```

Comment:
this is missing a `/`
should look like ${process.env.REACT_APP_LOCAL}/Api/Authentication/signUp